hhs_oauth_server/middleware.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.deprecation import MiddlewareMixin
+from rest_framework.exceptions import APIException
+from django.http import HttpResponse
+
+
+# Mixin for compatibility with Django <1.10
+class HandleAPIExceptionMiddleware(MiddlewareMixin):
```

Comment:
If you going to build a generic exception handler, can we make sure we're setting up a standard format that we're going to be happy having everywhere? There's a number of existing style guides that detail error response formats, and it would be good to follow one of them if we're expecting our developers to be able to get information out of our responses.

Reply:
Sure did you have on in mind? In this case the response from `oauth2_provider ` takes the form of `{error: <message>}`